components/gitpod-db/src/blocked-repository-db.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { BlockedRepository } from "@gitpod/gitpod-protocol/src/blocked-repositories-protocol";
+
+export const BlockedRepositoryDB = Symbol("BlockedRepositoryDB");
+
+export interface BlockedRepositoryDB {
+    isRepositoryBlocked(contextURL: string): Promise<BlockedRepository | undefined>;
```

Comment:
nit: don't want to get into bike shedding, but wanted to share that I was confused by a method name starting with `is...` to return an object. As a result, I had to read the code twice, and have a look at the unit tests to make sure I understand how it works.

Did you think about calling it `findRepositoryBlockedByUrl` or similar?

Reply:
Not bikeshedding 👍 I agree it needs changing.